distribution/changelog.txt

```diff
@@ -12,6 +14,8 @@
 - Improved: [#18970] Trying to load a non-park save will now display a context error.
 - Improved: [#18975] Add lift sprites for 60 steep hills on the wooden roller coaster.
 - Improved: [#19044] Added special thanks to RMC and Wiegand to the About page.
+- Improved: [#19131] Track missing objects when selecting scenery groups in console.
```

Comment:
I'm not sure end users are affected by this one too much. It just uses LOG_ERROR when selecting a scenery group, and I don't think that gets logged anywhere unless you start up a debug build from the command line. Maybe this one could be reworded to say "Log missing ..."

Reply:
You can see these logs if you run the console build (.com) that is distributed with all versions. I think therefore it does make sense to have this entry.